<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Deposit</class>
 <widget class="QDialog" name="Deposit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>941</width>
    <height>1082</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>941</width>
    <height>1082</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>941</width>
    <height>1082</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(50,50,50);</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <widget class="QLineEdit" name="sum_deposit">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>50</y>
     <width>200</width>
     <height>30</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>200</width>
     <height>1</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>200</width>
     <height>30</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Pixeloid Sans</family>
     <bold>true</bold>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">color: black; background-color: rgb(180, 179, 161); border: none;</string>
   </property>
   <property name="text">
    <string>100000</string>
   </property>
   <property name="maxLength">
    <number>255</number>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
   <property name="placeholderText">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="term_deposit">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>90</y>
     <width>200</width>
     <height>30</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>200</width>
     <height>30</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>200</width>
     <height>30</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Pixeloid Sans</family>
     <bold>true</bold>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">color: black; background-color: rgb(180, 179, 161); border: none;</string>
   </property>
   <property name="text">
    <string>12</string>
   </property>
   <property name="maxLength">
    <number>255</number>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="deposit_rate">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>130</y>
     <width>200</width>
     <height>30</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>200</width>
     <height>30</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>200</width>
     <height>30</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Pixeloid Sans</family>
     <bold>true</bold>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">color: black; background-color: rgb(180, 179, 161); border: none;</string>
   </property>
   <property name="text">
    <string>16</string>
   </property>
   <property name="maxLength">
    <number>255</number>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_1_sum_deposit">
   <property name="geometry">
    <rect>
     <x>33</x>
     <y>50</y>
     <width>220</width>
     <height>31</height>
    </rect>
   </property>
   <property name="sizeIncrement">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="baseSize">
    <size>
     <width>40</width>
     <height>0</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Eurostile</family>
     <pointsize>17</pointsize>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(170,170,170);</string>
   </property>
   <property name="text">
    <string>Сумма вклада, руб.</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2_term_deposit">
   <property name="geometry">
    <rect>
     <x>33</x>
     <y>90</y>
     <width>220</width>
     <height>31</height>
    </rect>
   </property>
   <property name="sizeIncrement">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="baseSize">
    <size>
     <width>40</width>
     <height>0</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Eurostile</family>
     <pointsize>17</pointsize>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(170,170,170);</string>
   </property>
   <property name="text">
    <string>Срок размещения, мес.</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3_percent_deposit">
   <property name="geometry">
    <rect>
     <x>33</x>
     <y>130</y>
     <width>220</width>
     <height>31</height>
    </rect>
   </property>
   <property name="sizeIncrement">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="baseSize">
    <size>
     <width>40</width>
     <height>0</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Eurostile</family>
     <pointsize>17</pointsize>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(170,170,170);</string>
   </property>
   <property name="text">
    <string>Процентная ставка, %</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="add_to_deposit">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>220</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Eurostile</family>
     <pointsize>17</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white;</string>
   </property>
   <property name="text">
    <string>Добавлять ко вкладу</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QRadioButton" name="pay_to_account">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>250</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Eurostile</family>
     <pointsize>17</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white;</string>
   </property>
   <property name="text">
    <string>Выплачивать на счёт</string>
   </property>
  </widget>
  <widget class="QPushButton" name="calculate_deposit">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>300</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>20</width>
     <height>20</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>680</width>
     <height>4800</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Eurostile</family>
     <pointsize>16</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	border-radius: 6px;
	color: white;
	background-color: rgb(126, 44, 61); 
}
QPushButton:pressed {
	background-color: rgb(146, 64, 81); 
}
QPushButton:hover {
	border: 1px solid rgb(146, 64, 81); 
}</string>
   </property>
   <property name="text">
    <string>Рассчитать</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="info_message">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>350</y>
     <width>650</width>
     <height>143</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>650</width>
     <height>143</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>650</width>
     <height>143</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Kefa</family>
     <pointsize>16</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white;
background-color: rgb(50,50,50);
border: none;</string>
   </property>
   <property name="cursorWidth">
    <number>0</number>
   </property>
  </widget>
  <widget class="QTableView" name="table_deposit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>499</y>
     <width>901</width>
     <height>561</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>901</width>
     <height>561</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>901</width>
     <height>561</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QTableView {gridline-color: #b0b0b0;} 
QHeaderView::section {color: black; background-color: white;
font-size: 14pt; border-style: none; border-bottom: 1px solid 
#b0b0b0; border-right: 1px solid #b0b0b0; padding-left: 5px;} 
QHeaderView::section:horizontal {border-top: 1px solid #b0b0b0;} 
QHeaderView::section:vertical {border-left : 1px solid #b0b0b0;} 
QTableView::item {background-color: #606060; color: white;}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::NoEditTriggers</set>
   </property>
   <attribute name="horizontalHeaderVisible">
    <bool>true</bool>
   </attribute>
   <attribute name="horizontalHeaderCascadingSectionResizes">
    <bool>false</bool>
   </attribute>
   <attribute name="horizontalHeaderMinimumSectionSize">
    <number>30</number>
   </attribute>
   <attribute name="horizontalHeaderHighlightSections">
    <bool>false</bool>
   </attribute>
   <attribute name="horizontalHeaderStretchLastSection">
    <bool>false</bool>
   </attribute>
   <attribute name="verticalHeaderVisible">
    <bool>false</bool>
   </attribute>
   <attribute name="verticalHeaderCascadingSectionResizes">
    <bool>false</bool>
   </attribute>
   <attribute name="verticalHeaderHighlightSections">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QLineEdit" name="tax_rate">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>170</y>
     <width>200</width>
     <height>30</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>200</width>
     <height>30</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>200</width>
     <height>30</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Pixeloid Sans</family>
     <bold>true</bold>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">color: black; background-color: rgb(180, 179, 161); border: none;</string>
   </property>
   <property name="text">
    <string>35</string>
   </property>
   <property name="maxLength">
    <number>255</number>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_4_tax_rate">
   <property name="geometry">
    <rect>
     <x>33</x>
     <y>170</y>
     <width>220</width>
     <height>31</height>
    </rect>
   </property>
   <property name="sizeIncrement">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="baseSize">
    <size>
     <width>40</width>
     <height>0</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Eurostile</family>
     <pointsize>17</pointsize>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(170,170,170);</string>
   </property>
   <property name="text">
    <string>Налоговая ставка, %</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="key_rate">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>210</y>
     <width>200</width>
     <height>30</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>200</width>
     <height>30</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>200</width>
     <height>30</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Pixeloid Sans</family>
     <bold>true</bold>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">color: black; background-color: rgb(180, 179, 161); border: none;</string>
   </property>
   <property name="text">
    <string>9.5</string>
   </property>
   <property name="maxLength">
    <number>255</number>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_5_key_rate">
   <property name="geometry">
    <rect>
     <x>33</x>
     <y>210</y>
     <width>220</width>
     <height>31</height>
    </rect>
   </property>
   <property name="sizeIncrement">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="baseSize">
    <size>
     <width>40</width>
     <height>0</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Eurostile</family>
     <pointsize>17</pointsize>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(170,170,170);</string>
   </property>
   <property name="text">
    <string>Ключевая ставка ЦБ, %</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6_frequency_payment">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>33</x>
     <y>250</y>
     <width>220</width>
     <height>31</height>
    </rect>
   </property>
   <property name="sizeIncrement">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="baseSize">
    <size>
     <width>40</width>
     <height>0</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Eurostile</family>
     <pointsize>17</pointsize>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(170,170,170);</string>
   </property>
   <property name="text">
    <string>Периодичность выплат</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7_capitalization">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>33</x>
     <y>250</y>
     <width>241</width>
     <height>31</height>
    </rect>
   </property>
   <property name="sizeIncrement">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="baseSize">
    <size>
     <width>40</width>
     <height>0</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Eurostile</family>
     <pointsize>17</pointsize>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(170,170,170);</string>
   </property>
   <property name="text">
    <string>Капитализация процентов</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9_refill">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>50</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="sizeIncrement">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="baseSize">
    <size>
     <width>40</width>
     <height>0</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Eurostile</family>
     <pointsize>17</pointsize>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(170,170,170);</string>
   </property>
   <property name="text">
    <string>Пополнение вклада</string>
   </property>
  </widget>
  <widget class="QComboBox" name="frequency_payment">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>250</y>
     <width>200</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>30</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>200</width>
     <height>300</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Pixeloid Sans</family>
     <pointsize>13</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QComboBox {combobox-popup: 1; color: #000; background-color: rgb(180, 179, 161); border: none; } 
QComboBox::item {background: rgb(65, 65, 65); color: rgb(200, 200, 200);} 
QComboBox::item:selected {background: rgb(126, 44, 61);}</string>
   </property>
   <property name="maxVisibleItems">
    <number>20</number>
   </property>
   <property name="frame">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QComboBox" name="frequency_capitalization">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>250</y>
     <width>200</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>200</width>
     <height>30</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>200</width>
     <height>300</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Pixeloid Sans</family>
     <pointsize>13</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QComboBox {combobox-popup: 1; color: #000; background-color: rgb(180, 179, 161); border: none; } 
QComboBox::item {background: rgb(65, 65, 65); color: rgb(200, 200, 200);} 
QComboBox::item:selected {background: rgb(126, 44, 61);}</string>
   </property>
   <property name="maxVisibleItems">
    <number>20</number>
   </property>
   <property name="frame">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QDateEdit" name="date_start_deposit">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>90</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Eurostile</family>
     <pointsize>15</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(50,50,50);
border-color: rgb(50,50,50);
border-width: 1px;
border-style: none;
color: white;</string>
   </property>
   <property name="frame">
    <bool>false</bool>
   </property>
   <property name="buttonSymbols">
    <enum>QAbstractSpinBox::PlusMinus</enum>
   </property>
   <property name="currentSection">
    <enum>QDateTimeEdit::DaySection</enum>
   </property>
   <property name="displayFormat">
    <string>dd.MM.yyyy</string>
   </property>
  </widget>
  <widget class="QComboBox" name="frequency_refill">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>90</y>
     <width>200</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>200</width>
     <height>30</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>200</width>
     <height>300</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Pixeloid Sans</family>
     <pointsize>13</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QComboBox {combobox-popup: 1; color: #000; background-color: rgb(180, 179, 161); border: none; } 
QComboBox::item {background: rgb(65, 65, 65); color: rgb(200, 200, 200);} 
QComboBox::item:selected {background: rgb(126, 44, 61);}</string>
   </property>
   <property name="maxVisibleItems">
    <number>20</number>
   </property>
   <property name="frame">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="sum_refill">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>130</y>
     <width>200</width>
     <height>30</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>200</width>
     <height>30</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>200</width>
     <height>30</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Pixeloid Sans</family>
     <bold>true</bold>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">color: black; background-color: rgb(180, 179, 161); border: none;</string>
   </property>
   <property name="text">
    <string>10000</string>
   </property>
   <property name="maxLength">
    <number>255</number>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
   <property name="placeholderText">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_8_date_start_deposit">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>50</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="sizeIncrement">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="baseSize">
    <size>
     <width>40</width>
     <height>0</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Eurostile</family>
     <pointsize>17</pointsize>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(170,170,170);</string>
   </property>
   <property name="text">
    <string>Дата начала срока</string>
   </property>
  </widget>
  <zorder>table_deposit</zorder>
  <zorder>sum_deposit</zorder>
  <zorder>term_deposit</zorder>
  <zorder>deposit_rate</zorder>
  <zorder>label_1_sum_deposit</zorder>
  <zorder>label_2_term_deposit</zorder>
  <zorder>label_3_percent_deposit</zorder>
  <zorder>add_to_deposit</zorder>
  <zorder>pay_to_account</zorder>
  <zorder>calculate_deposit</zorder>
  <zorder>info_message</zorder>
  <zorder>tax_rate</zorder>
  <zorder>label_4_tax_rate</zorder>
  <zorder>key_rate</zorder>
  <zorder>label_5_key_rate</zorder>
  <zorder>label_6_frequency_payment</zorder>
  <zorder>label_7_capitalization</zorder>
  <zorder>label_9_refill</zorder>
  <zorder>frequency_payment</zorder>
  <zorder>frequency_capitalization</zorder>
  <zorder>date_start_deposit</zorder>
  <zorder>frequency_refill</zorder>
  <zorder>sum_refill</zorder>
  <zorder>label_8_date_start_deposit</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
